File: DeleteConfirmation.cs
using System;
using System.Data;
using System.Windows.Forms;

namespace MilkTeaOrderAndInventorySystem
{
    public partial class DeleteConfirmation
    {
        public string passAddress { get; set; }
        public string passConfirmation { get; set; }

        private const int CS_DROPSHADOW = 131072;

        protected override CreateParams CreateParams
        {
            get
            {
                var cp = base.CreateParams;
                cp.ClassStyle = cp.ClassStyle | CS_DROPSHADOW;
                return cp;
            }
        }

        public DeleteConfirmation()
        {
            InitializeComponent();
        }

        private void btnCancel_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void btnDelete_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.OK;
        }

        private void DeleteConfirmation_Load(object sender, EventArgs e)
        {
            if (UniversalVariables.conn.State == ConnectionState.Open)
            {
                UniversalVariables.conn.Close();
            }
            UniversalVariables.conn.ConnectionString = "server=localhost;user id=root;port=3306;password=0102bryan;database=accountdb";

            if (passConfirmation != default)
            {
                lblConfirmation.Text = passConfirmation;
            }
            else
            {
                var obj = new Homepage();
                obj.Hide();
            }
        }
    }
}

File: DeleteConfirmation.vb
Public Class DeleteConfirmation
    Public Property passAddress As String
    Public Property passConfirmation As String

    Private Const CS_DROPSHADOW As Integer = 131072

    Protected Overrides ReadOnly Property CreateParams() As System.Windows.Forms.CreateParams
        Get
            Dim cp As CreateParams = MyBase.CreateParams
            cp.ClassStyle = cp.ClassStyle Or CS_DROPSHADOW
            Return cp
        End Get
    End Property

    Private Sub btnCancel_Click(sender As Object, e As EventArgs) Handles btnCancel.Click
        Me.Close()
    End Sub

    Private Sub btnDelete_Click(sender As Object, e As EventArgs) Handles btnDelete.Click
        Me.DialogResult = DialogResult.OK
    End Sub

    Private Sub DeleteConfirmation_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        If conn.State = ConnectionState.Open Then
            conn.Close()
        End If
        conn.ConnectionString = "server=localhost;user id=root;port=3306;password=0102bryan;database=accountdb"

        If Not passConfirmation = Nothing Then
            lblConfirmation.Text = passConfirmation
        Else
            Dim obj As New Homepage
            obj.Hide()
        End If
    End Sub
End Class